posix.mak

```diff
@@ -279,7 +284,13 @@ install :
 	$(MAKE) -f $(MAKEFILE) OS=$(OS) MODEL=$(MODEL) BUILD=release INSTALL_DIR=$(INSTALL_DIR) \
 		DMD=$(DMD) install2
 
+ifeq (1,$(BUILD_WAS_SPECIFIED))
 unittest : $(addsuffix .d,$(addprefix unittest/,$(D_MODULES)))
+else
+unittest : unittest-debug unittest-release
+unittest-%: 
+	$(MAKE) -j4 unittest -f $(MAKEFILE) OS=$(OS) MODEL=$(MODEL) DMD=$(DMD) BUILD=$*
```

Comment:
On Wed, Feb 04, 2015 at 09:34:32PM -0800, Brad Roberts wrote:
[...]
> Don't specify -j<anything> inside makefiles like this.  Those should
> be (and are in the case of the auto-testers) controlled from the
> originating outer-most make.
[...]

Actually, I would rather we don't specify `-j` *at all*, because it will
overwhelm older machines, and under-utilize newer ones. Just let the
user specify it themselves. The autotester can probably be configured to
do this automatically, with a value of `-j` chosen according to the
current host.